build/build.sh

```diff
@@ -32,10 +34,10 @@ pandoc --verbose \
   --output=output/index.html \
   $INPUT_PATH
 
-# Create PDF outpout
+# Create PDF output
 echo "Exporting PDF manuscript"
 pandoc \
-  --from=markdown \
+  --from=markdown+yaml_metadata_block \
```

Comment:
As per http://pandoc.org/MANUAL.html, I believe the `yaml_metadata_block` is on by default.